Queryable.go

```diff
@@ -0,0 +1,17 @@
+package dblib
```

Comment:
uppercase filenames? is that typical in Go?

Reply:
nope. I haven't written Java in 10 years, but it's still the way I naturally write code.